flow/network.cpp

```diff
@@ -123,7 +146,11 @@ std::vector<NetworkAddress> NetworkAddress::parseList(std::string const& addrs)
 }
 
 std::string NetworkAddress::toString() const {
-	return formatIpPort(ip, port) + (isTLS() ? ":tls" : "");
+	std::string ipString = formatIpPort(ip, port) + (isTLS() ? ":tls" : "");
+	if (fromHostname) {
+		return ipString + "(fromHostname)";
```

Comment:
_Late to the party._

If we were to append something to indicate a `NetworkAddress` from a hostname, will it be helpful to append the original hostname too here? some ting like `(fromHostname:blah:1234:tls)`? @brownleej thoughts?

Reply:
That was my first thought too. However there can't be any non-trivially-destructed members (in our case, string) in NetworkAddress.